apiserver/facades/client/modelconfig/modelconfig.go

```diff
@@ -194,6 +219,33 @@ func (c *ModelConfigAPI) ModelSet(ctx context.Context, args params.ModelSet) err
 	return err
 }
 
+// setAgentStream is responsible for setting the agent stream to use on the
+// current model. This exists because the way ask users to control this value is
+// still via model config as a user interface. If the value of agent stream
+// passed to this function is an empty string no operation will be performed.
+func (s *ModelConfigAPI) setAgentStream(ctx context.Context, agentStream string) error {
+	if agentStream == "" {
```

Comment:
So the agent stream can not be cleared once it's set, right?

Reply:
Correct. We always have a value.